.repoman.yml

```diff
@@ -43,6 +43,17 @@ issues:
       pass:
         - labels-add: ["source incompatible"]
 
+    # Checks breaking change version and adds a label
+    - check:
+        - type: query
+          value: "contains(Issue.body, '.NET 6 Preview 7') == `true` 
```

Comment:
I believe YAML needs special syntax to make mutliline strings. I'm also not sure if JMESPath handles multiline.

But why not just use a simple:

```
contains(Issue.body, '.NET 6 Preview') == `true`
```

Reply:
We can probably just use the simple form. I'll do another commit.